### DISCUSSION TITLE
Cross-Section data

### DISCUSSION THREAD

**Discussion by 5ub-Man** [scikit-hep/pylhe#216] [Q&A]
I'm using pylhe for the first time and I can't figure out how to read the cross-section, error of each event from a lhe file generated through madgraph.

**Comment by eduardo-rodrigues** [comment]
Hello. Let me ping @matthewfeickert and @lukasheinrich just in case. Not all info on LHE files is made available through the package at tghe moment. What you ask is one example. It should not be too difficult to extend. Would you be interested in contributing with a PR for that enhancement?
